app/scripts/scripts.js

```diff
@@ -1,17 +1,28 @@
-/* For picking letters from alphabet keypad. Returns the
-   letter chosen or "already picked"
-*/
-function pickLetter(letter) {
+(function () {
   "use strict";
-  var result;
-
-  if (letter.className === "letter-button") {
-    result = letter.innerHTML;
-    letter.className = "letter-disabled";
-  } else {
-    result = "already picked";
+  
+  // add pickLetter to all buttons on alphabet keypad
+  $(".letter-button").on("click", function () {
+    pickLetter(this);
+  });
+  
+  // pick a letter from alphabet keypad
+  function pickLetter(button) {
+    var letterChosen;
+    
+    if (button.className === "letter-button") {
```

Comment:
The idea was it's always true in the beginning. After we press a button, we change its class to "letter-disabled" and this condition will be false. Is there a better way to organize this?